tests/driver_ads101x/Makefile

```diff
@@ -0,0 +1,15 @@
+include ../Makefile.tests_common
+
+FEATURES_REQUIRED = periph_i2c
+
+USEMODULE += ads101x
+
+# set alert parameters in case they are undefined
+TEST_ADS101X_LOW_LIMIT     ?= 10000
```

Comment:
Why not directly use those values in the ads101x_params.h driver file ?

Reply:
Good question. The answer is... no reason except that the adcxx1c driver did it that way. I have put them in ads101x_params.h